src/lib/functions/runtimes/js/builders/zisi.js

```diff
@@ -96,14 +103,6 @@ module.exports = async ({ config, directory, errorExit, func, projectRoot }) =>
     functionsConfig['*'].nodeBundler = 'esbuild'
   }
 
-  // TODO: Resolve functions config globs so that we can check for the bundler
-  // on a per-function basis.
-  const isUsingEsbuild = ['esbuild_zisi', 'esbuild'].includes(functionsConfig['*'].nodeBundler)
-
-  if (!isUsingEsbuild) {
```

Comment:
ISC needs to parse every single file, so sadly there's no way around ZISI :/

Reply:
reverted, now using `listFunction`! https://github.com/netlify/cli/pull/3689/commits/77ef9726b5514502b6962400b653d9ef8db2087a